### DISCUSSION TITLE
Unable To Login

### DISCUSSION THREAD

**Discussion by qasimali1st** [community/community#139465] [Other Feature Feedback, Questions, & Ideas] [answered]
I Got To Reset My Mobile And Erased My All Data. Now I Am Trying To Login To Github But It's Asking For Device Verification, But The Case Is That I Don't Have Access To My Email, Not Added Any Recovery Method. Just I Remember The Password And Username Of My Github Account. Any Other Way To Login?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @qasimali1st ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
